### PR TITLE
Translation of triptych pt-br

### PR DESCRIPTION
**Pull request by botoserelepe**
@lh1008 here is the translation.

pinging @netrik182 @nelsonrenan for review.

### CODE REVIEW

File: monero-outreach-docs/translations/pt-br/archive/triptych_pt-br.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+O movimento de Monero entre as chaves públicas é mantido em segredo agora usando o que são chamadas de assinaturas criptográficas em anel. Com essa abordagem, a chave pública para uma TXO real é escondida colocando-a junto com chamarizes (também chamados de mixins). Olhando para a transação, não há maneira de dizer se a TXO real está sendo usada ou se é algum dos chamarizes. Você pode ver exemplos de entradas e saídas de TXOs olhando uma transação em um explorador da blockchain do Monero, como [moneroblocks.info](https://moneroblocks.info/) ou [xmrchain.net](https://xmrchain.net/).
```
Comment (netrik182) [inline monero-outreach-docs/translations/pt-br/archive/triptych_pt-br.md:9]: Let me know what you think about this suggestion below. 

```suggestion
Hoje, o movimento de Monero entre as chaves públicas é mantido em segredo usando um mecanismo chamado de assinaturas criptográficas em anel. Com essa abordagem, a chave pública para uma TXO real é escondida colocando-a junto com chamarizes (também chamados de mixins). Olhando para a transação, não há maneira de dizer se a TXO real está sendo usada ou se é algum dos chamarizes. Você pode ver exemplos de entradas e saídas de TXOs olhando uma transação em um explorador da blockchain do Monero, como [moneroblocks.info](https://moneroblocks.info/) ou [xmrchain.net](https://xmrchain.net/).
```

File: monero-outreach-docs/translations/pt-br/archive/triptych_pt-br.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+Os algoritmos fundamentais das assinaturas em anel também melhoraram. O whitepaper original do CryptoNote fornecia um esquema de assinatura em anel diferente da implementação atual mais eficiente baseada em assinaturas de Grupo Anônimo Espontâneo Multicamadas Conectável (MLSAG, do inglês Multilayered Linkable Spontaneous Anonymous Group). Uma melhoria muito esperada, chamada de assinaturas em Grupo Anônimo Espontâneo Conciso Conectável (CLSAG, do inglês Concise Linkable Spontaneous Anonymous Group), virá em breve e espera-se que ela reduza o tamanho total da uma transação para 1/4 e o tempo de verificação em aproximadamente 1/10 (para uma transação de 2 entradas e 2 saídas).
```
Comment (netrik182) [inline monero-outreach-docs/translations/pt-br/archive/triptych_pt-br.md:13]:
```suggestion
Os algoritmos fundamentais das assinaturas em anel também melhoraram. O whitepaper original do CryptoNote fornecia um esquema de assinatura em anel diferente da implementação atual, mais eficiente, que basea-se em assinaturas de Grupo Anônimo Espontâneo Multicamadas Conectável (MLSAG, do inglês Multilayered Linkable Spontaneous Anonymous Group). Uma melhoria muito esperada, chamada de assinaturas em Grupo Anônimo Espontâneo Conciso Conectável (CLSAG, do inglês Concise Linkable Spontaneous Anonymous Group), virá em breve e espera-se que ela reduza o tamanho total da uma transação para 1/4 e o tempo de verificação em aproximadamente 1/10 (para uma transação de 2 entradas e 2 saídas).
```

File: monero-outreach-docs/translations/pt-br/archive/triptych_pt-br.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+
+Todas essas melhorias deram ao Monero um anonimato poderoso. Toda vez que se gasta Moneros, cada fonte de TXO real na forma de TXO de chave pública, já criptograficamente desconectada do endereço Monero do remetente, é escondida entre 10 chamarizes de chave pública que são escolhidos para parecerem possibilidades reais. Isso é bastante frustrante para a fiscalização, tudo enquanto o software Monero roda rápida e facilmente, dando um tamanho e velocidade de blockchain plausíveis.
+
+### _Desafios_
+
+Porém, a ocultação do remetente de Monero não é perfeita. Dez chamarizes nem sempre são suficientes porque alguns TXOs aleatórios podem ser identificáveis. Eles podem já ter sido gastos anteriormente com conhecimento público. A existência de listas de excluídos mostra que isso é um problema potencial e dá uma alternativa àqueles que têm motivação e iniciativa para adicionar a lista às suas carteiras. E existem outras formas de identificar chamarizes, incluindo vincular TXOs à pools e corretoras, estimando a sua validade em outras transações, e usando probabilidades de timing.
```
Comment (netrik182) [inline monero-outreach-docs/translations/pt-br/archive/triptych_pt-br.md:19]: Maybe it's better to explain wait "timing" means here. See my suggestion:

```suggestion
Porém, a ocultação do remetente de Monero não é perfeita. Dez chamarizes nem sempre são suficientes porque alguns TXOs aleatórios podem ser identificáveis. Eles podem já ter sido gastos anteriormente com conhecimento público. A existência de listas de excluídos mostra que isso é um problema potencial e dá uma alternativa àqueles que têm motivação e iniciativa para adicionar a lista às suas carteiras. E existem outras formas de identificar chamarizes, incluindo vincular TXOs à pools e corretoras, estimando a sua validade em outras transações, e usando análises probabilísticas de timing (que correlaciona o tempo/hora das transações).
```

File: monero-outreach-docs/translations/pt-br/archive/triptych_pt-br.md [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+
+Imagine uma formação policial contra um injustamente acusado e mais 10 chamarizes. Se os chamarizes são conhecidamente de falsas testemunhas, isso não funcionaria para proteger o acusado. O mesmo vale para assinaturas em anel do Monero: esconder entre chamarizes só funciona quando os chamarizes são estranhos, e para adversários, adversários particularmente habilidosos, apenas 10 chamarizes pode ser inadequado. Então por que apenas não usar mais?
+
+Bem, para o algoritmo MLSAG/CLSAG usado pelo Monero, tanto o tamanho da transação quando o tempo de processamento aumenta linearmente com o número de chamarizes. Usar 10 chamarizes balanceia a necessidade de privacidade com a necessidade de uma blockchain menor e um tempo de processamento maior. Aumentar o número de chamarizes ajudaria na privacidade, mas também inflaria o tamanho da transação de Monero, que já é algumas vezes maior que a do Bitcoin, e atolar as carteiras que estão fazendo validação.
```
Comment (netrik182) [inline monero-outreach-docs/translations/pt-br/archive/triptych_pt-br.md:23]:
```suggestion
Bem, para o algoritmo MLSAG/CLSAG usado pelo Monero, tanto o tamanho da transação quando o tempo de processamento aumenta linearmente com o número de chamarizes. Usar 10 chamarizes balanceia a necessidade de privacidade com a necessidade de uma blockchain menor e um tempo de processamento mais rápido. Aumentar o número de chamarizes ajudaria na privacidade, mas também inflaria o tamanho da transação do Monero, que já é algumas vezes maior que a do Bitcoin, e atolaria as carteiras que estão fazendo validação.
```

File: monero-outreach-docs/translations/pt-br/archive/triptych_pt-br.md [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+
+### _Triptych_
+
+Enfrentando esses desafios está o Triptych, um novo algoritmo para assinaturas em anel desenvolvido por Sarang Noether e Brandon Goodell dos Laboratórios de Pesquisa Monero (MRL, do inglês Monero Research Labs) com ajuda do parceiro externo Arthur Blue (RandomRun). Com o Triptych, o tamanho da assinatura em anel cresce logaritmicamente em vez de linearmente. O crescimento logarítmico é muito devagar, o melhor padrão de crescimento sem precisar confiar em alguém para usar e depois destruir a aleatoriedade durante a configuração (como foi feito pela Zcash). O tempo de verificação ainda cresce linearmente com o número de chamarizes no Triptych, mas a verificação não pode ser em lotes e avaliada usando algoritmos especiais, os quais fornecem maior velocidade.
```
Comment (netrik182) [inline monero-outreach-docs/translations/pt-br/archive/triptych_pt-br.md:27]:
```suggestion
Enfrentando esses desafios está o Triptych, um novo algoritmo para assinaturas em anel desenvolvido por Sarang Noether e Brandon Goodell dos Laboratórios de Pesquisa Monero (MRL, do inglês Monero Research Labs) com ajuda do parceiro externo Arthur Blue (RandomRun). Com o Triptych, o tamanho da assinatura em anel cresce logaritmicamente em vez de linearmente. O crescimento logarítmico é muito devagar, o melhor padrão de crescimento sem precisar confiar em alguém para usar e depois destruir a aleatoriedade durante a configuração (como foi feito pela Zcash). O tempo de verificação ainda cresce linearmente com o número de chamarizes no Triptych, mas a verificação pode ser feita em lote e avaliada usando algoritmos especiais, os quais fornecem maior velocidade.
```

File: monero-outreach-docs/translations/pt-br/archive/triptych_pt-br.md [hunk 6]
```diff
@@ -0,0 +1,41 @@
...
+
+O Triptych supera vários desafios de assinaturas em anel aplicadas ao Monero. Ele fornece capacidade de conexões. A técnica de assinaturas em anel do Monero precisa bloquear a reutilização de chaves públicas legítimas de forma a prevenir que um TXO seja gasto mais de uma vez (o tão chamado gasto duplo). O Tritych faz isso usando rótulos de associação que são similares às atuais imagens chave. O Triptych também reforça o uso de transações confidenciais para esconder quantidades de Monero. Dados especiais chamados de chaves de comprometimento permitem a comprovação de que os valores do Monero em uma transação são legítimos, sem divulgar esses valores. Essas chaves de comprometimento com as chaves de assinatura e os rótulos de associação são três componentes que dão origem ao nome Triptych, significando algo em três partes.
```
Comment (netrik182) [inline monero-outreach-docs/translations/pt-br/archive/triptych_pt-br.md:29]: A slight and optional change but makes sense, I think.

```suggestion
O Triptych supera vários desafios de assinaturas em anel aplicadas ao Monero. Ele fornece capacidade de conexões. A técnica de assinaturas em anel do Monero precisa bloquear a reutilização de chaves públicas legítimas de forma a prevenir que um TXO seja gasto mais de uma vez (o tão chamado gasto duplo). O Tritych faz isso usando rótulos de associação que são similares às atuais imagens chave. O Triptych também reforça o uso de transações confidenciais para esconder quantidades de Monero. Dados especiais chamados de chaves de comprometimento permitem a comprovação de que os valores do Monero em uma transação são legítimos, sem divulgar esses valores. Essas chaves de comprometimento com as chaves de assinatura e os rótulos de associação são três componentes que dão origem ao nome Triptych, significando algo que possui três partes.
```

File: monero-outreach-docs/translations/pt-br/archive/triptych_pt-br.md [hunk 7]
```diff
@@ -0,0 +1,41 @@
...
+
+### _Teste e alternativas_
+
+O Triptych é executado de forma rápida na prática. Um dos seus criadores, Sarang Noether, relatou um código de teste que melhorou consideravelmente o tempo de execução. Aproximadamente, usar 63 chamarizes com Tritych roda 10% mais rápido que usar 10 chamarizes com CLSAG. Esses resultados indicam uma melhoria fundamental para o Monero que também serão praticadas em campo.
+
+Mas ainda tem trabalho, já que é necessária uma validação rigorosa para novos algoritmos como o Triptych entrarem no código base do Monero. Mesmo que o paper do algoritmo Triptych já tenha sido revisado (e aceito no ESORICS CBT 2020 Workshop), o Tritych ainda pode ser melhorado com modificações nos algoritmos e no software que implementa esses algoritmos. E a implementação pode ser ainda mais otimizada para reduzir o tempo de validação. Gastar esforços com o Triptych substitui a necessidade de trabalhar em alternativas.
```
Comment (netrik182) [inline monero-outreach-docs/translations/pt-br/archive/triptych_pt-br.md:35]:
```suggestion
Mas ainda existe trabalho a ser feito, já que é necessária uma validação rigorosa para novos algoritmos como o Triptych entrarem no código base do Monero. Mesmo que o paper do algoritmo Triptych já tenha sido revisado (e aceito no ESORICS CBT 2020 Workshop), o Tritych ainda pode ser melhorado com modificações nos algoritmos e no software que implementa esses algoritmos. E a implementação pode ser ainda mais otimizada para reduzir o tempo de validação. Gastar esforços com o Triptych substitui a necessidade de trabalhar em alternativas.
```

**Review by netrik182 — CHANGES_REQUESTED** [review]
Very nice work @botoserelepe. This text is somewhat technical but you mastered it. Congrats!

I added a few suggestions and corrections. Please take a look and let me know if you don't agree with them.

**Comment by lh1008** [comment]
Hello,

@botoserelepe, @netrik182 did some suggestions to the translation. Pinging you so you can take a look at them.

**Comment by botoserelepe** [comment]
@lh1008 corrections were done!
@netrik182 thanks for the review, I accepted all your suggestions. :)

**Review by lh1008 — APPROVED** [review]
Hey @botoserelepe @netrik182 

Great work, thank you :). Everything looks good, approved.

**Pull request merged**